server/integration_tests/standalone/adversarial.py

```diff
@@ -74,6 +74,10 @@ class Mode:
     "results_csv_file", None,
     f"The results csv file to be inspected in {Mode.COMPUTE_FILE_STATS} mode.")
 
+flags.DEFINE_boolean(
+    "use_sdg", False,
```

Comment:
Can we make this an enum instead of a boolean? We can call the flag `dc` (or something similar) with `main` and `sdg` as valid values. This will make it easier to support more custom DCs in the future (basically add an enum value).

Reply:
Made it a string which can be supplied to the dc= param.